<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoQueryDialog</class>
 <widget class="QDialog" name="InfoQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>查询条件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_ConditionType1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ConditionEdit1"/>
       </item>
       <item>
        <widget class="QDateEdit" name="m_ConditionData1">
         <property name="displayFormat">
          <string>yyyy/M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddCondition1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>增加条件</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="m_ConditionType2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ConditionEdit2"/>
       </item>
       <item>
        <widget class="QDateEdit" name="m_ConditionData2">
         <property name="displayFormat">
          <string>yyyy/M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddCondition2">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>增加条件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_DelCondition2">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>删除条件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="m_ConditionType3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ConditionEdit3"/>
       </item>
       <item>
        <widget class="QDateEdit" name="m_ConditionData3">
         <property name="displayFormat">
          <string>yyyy/M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddCondition3">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>增加条件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_DelCondition3">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>删除条件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="m_ConditionType4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ConditionEdit4"/>
       </item>
       <item>
        <widget class="QDateEdit" name="m_ConditionData4">
         <property name="displayFormat">
          <string>yyyy/M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddCondition4">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>增加条件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_DelCondition4">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>删除条件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="m_ConditionType5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ConditionEdit5"/>
       </item>
       <item>
        <widget class="QDateEdit" name="m_ConditionData5">
         <property name="displayFormat">
          <string>yyyy/M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_DelCondition5">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>删除条件</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>人事号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>姓名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>专业</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类别</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>用户详情</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>编辑</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>删除</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>导出Excel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
